src/main/java/betterquesting/client/gui2/GuiHome.java

```diff
@@ -65,36 +66,43 @@ public GuiHome(GuiScreen parent)
 	}
 	
 	@Override
-	public void initPanel()
-	{
+	public void initPanel() {
 		super.initPanel();
-		
+
+		GuiHome.bookmark = this;
+		// If we come to home gui - we set skip home to false
+		if (BQ_Settings.skipHome) {
+			ConfigHandler.config.get(Configuration.CATEGORY_GENERAL, "Skip Home", false).set(false);
+			ConfigHandler.config.save();
+			BQ_Settings.skipHome = false;
+		}
```

Comment:
I am not really sure why this is needed. Why would we update the value of skipHome if the user manually moves to the main menu? I would personally expect this to not be updated, so that if I go to the main menu to make some changes to the theme or party and close my book, I would be back at the quest page, rather than having to manually go to the quests page